Multisig/UI/Settings/OwnerKeyManagement/LedgerOwnerKey/LedgerKeyPickerViewController.swift

```diff
@@ -48,9 +48,30 @@ class LedgerKeyPickerViewController: SegmentViewController {
 
     @objc func didTapImport() {
         guard let selectedIndex = selectedIndex,
-              let controller = viewControllers[selectedIndex] as? LedgerKeyPickerContentViewController else { return }
-        controller.importSelectedKey()
-        completion()
+              let controller = viewControllers[selectedIndex] as? LedgerKeyPickerContentViewController,
+              let key = controller.selectedKey else { return }
+
+        var namePrefix = ""
+        switch controller.keyType {
+        case .ledger: namePrefix = "Ledger key #"
+        case .ledgerLive: namePrefix = "Ledger Live key #"
+        }
+        let defaultName = "\(namePrefix)\(key.index + 1)"
```

Comment:
Do we need to have default name? 

I would be up to remove it in order to have same consistency with other import key types

Reply:
Here we can offer a good one, in other places, it is harder to do.
Now WalletConnect keys connected with default name even without this screeen